### ISSUE TITLE
pugjs.org misses an introduction / motivation

### ISSUE THREAD

**Issue by RoelRoel** [pugjs/pug#3271]
Some basic questions are not answered on the main page of your website:

What is pug?
Why use pug?

It just begins with installing it. I personally find it very annoying if frameworks / tools lack this simple introduction.

**Comment by trasherdk** [comment]
Because:
1.  You are not randomly discovering this thing.
  If you did randomly discover this thing, keep researching.
  While researching, not only will you expand your knowledge, you might even discover new stuff.

2. Reading the manual will get you far

**Comment by Bikeman868** [comment]
That is a very rude, arrogant and unnecessary response.
I agree with RoelRoel, this type of high level big picture really helps provide context when picking up something new and it would be a very quick and easy thing to do for someone that understands the library.

**Comment by trasherdk** [comment]
Damn. Now I have to go back to sensitivity class.

**Comment by camachoaleMX** [comment]
I agree with RoelRoel because if the page gives us an introduction we can think about how can use PUG and why can we should use it. But yes, some people only want to know how to install and let's go.
